src/jobsautoreport/report.py

```diff
@@ -24,9 +24,14 @@ def get_number_of_jobs_triggered(
 
     def get_average_duration_of_jobs(
         self, from_date: datetime, to_date: datetime
-    ) -> float:
+    ) -> tuple[int, int]:
```

Comment:
Return a  `timedelta` instead in will allow you to extract the information you need https://docs.python.org/3/library/datetime.html#datetime.timedelta